### ISSUE TITLE
UI for configuring bot settings and commands

### ISSUE THREAD

**Issue by FrankFlamme** [FrankFlamme/TwitchBot#2]
We need a UI for configuring the bot settings and commands in the database.

**Comment by Zerotask** [comment]
I'd separate the core of the bot from the presentation layer, so you can easily swap the UI, respectively create new UIs, e. g. Website with Blazor or GUI with WinForms.
This wouldn't enforce a user to host the bot on a website for example.

**Comment by FrankFlamme** [comment]
Good point. We separate the bot from the UI and start with creating a WinForms (.NET Core) application.

**Comment by Zerotask** [comment]
An idea would be to make the (core) bot a NuGet package.
